rules-tests/Renaming/Rector/Name/RenameClassRector/FixtureAutoImportNames/class_with_same_name_inside_different_namespace.php.inc

```diff
@@ -24,7 +24,7 @@ class Foo2 extends FooBar
     /**
      * @param DateTime $foo
      *
-     * @return Storage|\Illuminate\Support\Facades\Storage
+     * @return Foo2\Storage|\Illuminate\Support\Facades\Storage
```

Comment:
Ooops, this seems wrong, see https://3v4l.org/1qq8K

Reply:
Fixed 🎉 https://github.com/rectorphp/rector-src/pull/3149/commits/211bf0e6f65817a91781a612e666ec13b33daf77